src/objects.rs

```diff
@@ -1,5 +1,24 @@
 use collection::{Id, Idx};
 
+pub type CodesT = Vec<(String, String)>;
```

Comment:
Could be nice to have a comment on the choice of the structure, so someone smart in the future doesn't do the same mistakes/tests with other structures